### DISCUSSION TITLE
Cabagges dont get picked up and replanted by farmers

### DISCUSSION THREAD

**Discussion by AliSultanbekov** [vectorwing/FarmersDelight#875] [Q&A]
![image](https://github.com/vectorwing/FarmersDelight/assets/37249669/c7112b6a-51db-46fb-9fce-2cdf12b26ed8)
fix pls

**Comment by AliSultanbekov** [comment]
![plsfix](https://github.com/vectorwing/FarmersDelight/assets/37249669/5fca3188-58e0-4f19-b654-7753219b197a)

**Comment by BarchamMal** [comment]
Umm, why is this still even here? This is an *Issue* not a suggestion.

**Comment by vectorwing** [comment]
Thanks for the pointer!
Yeah, this is more of an issue indeed. Sadly, it's difficult to make Villagers interact with modded crops, as their selection of targets is hardcoded. Adding this compatibility is in the backlog.

**Comment by BarchamMal** [comment]
Well, I suppose this will fix itself once you get to Minecraft version 1.20, because then it is an itemTag, so theres no need to worry.

**Comment by vectorwing** [comment]
Thank you yet again for another pointer! 😆

I wasn't aware the `villager_plantable_seeds` tag existed on 1.20.1, which makes things a bit easier. I went ahead and implemented the feature above; now farmers can handle FD crops like vanilla ones.

The only exception to the norm is Rice: villagers can't plant the seed (due to bugs related to pathfinding/placement rules), but they'll still be able to harvest the top half of rice as it grows.

**Reply by BarchamMal** [reply]
Wonderful! Glad to help, and love this mod @Vectorwing!
